initialscreen.php

```diff
@@ -0,0 +1,256 @@
+$picture_data4 = retrieve_data_project($db, 4);
+$picture_data5 = retrieve_data_project($db, 5);
+$picture_data6 = retrieve_data_project($db, 6);
+$picture_data7 = retrieve_data_project($db, 7);
+$picture_data8 = retrieve_data_project($db, 8);
+$picture_data9 = retrieve_data_project($db, 9);
+
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" type="text/css" href="initialscreen.css">.
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:200,500">
+</head>
+<body>
+<!-- HOME PAGE -->
+<div class="headerbox">
+    <div class="applicationimages1">
+        <img src="html5-original-wordmark.svg" class="logoleft">
+        <img src="css3-original-wordmark.svg" class="logoleft">
+        <img src="javascript-original.svg" class="logoleft">
+    </div>
+    <div class="descriptionbox">
+        <div class="descrip-holder">
+            <h1 class="name">
+                Felix Gate
+            </h1>
+            <h2 class="title">
+                full stack developer, website creator
+            </h2>
+            <div class="downcontainer">
+                <a href="#project" >
+                    <p class="downlink">
+                        explore my work &#8628;
+                    </p>
+                </a>
+            </div>
+         </div>
+    </div>
+    <div class="applicationimages2">
+        <img src="php-original.svg" class="logoright">
+        <img src="git-original-wordmark.svg" class="logoright">
+        <img src="mysql-original-wordmark.svg" class="logoright">
+    </div>
+</div>
+<!-- SITCKY-NAV -->
+<div id="portfolio"></div> <!-- portfolio div required for internal links -->
+<nav class="stickynav">
+    <div class="linkcontainer" >
+        <a href="#" class="navlink">
+            home
+        </a>
+        <a href="#portfolio" class="navlink">
+            aboutme
+        </a>
+        <a href="#project" class="navlink">
+            porfolio
+        </a>
+        <a href="#contactme" class="navlink" >
+            contact me
+        </a>
+    </div>
+</nav>
+<!-- ABOUT ME CONTENT -->
+<div class="content-container">
+    <section class="aboutme" >
+       <div class="title-wrapper">
+            <h2 id="metitle">
+                About me
+            </h2>
+       </div>
+        <div id="my-photo-holder">
+            <img src="<?php echo $start_data['picture_link'] ?>" >
+        </div>
+        <p id="about-me-content">
+            <?php
+            echo $start_data['content']
+            ?>
+        </p>
+    </section>
+    <!--PORTFOLIO CONTENT    THIS IS WHERE THE PICTURE DATA WILL GO NEED TO BUILD HTML FOR THIS   -->
+    <div id="project"></div>
+    <section class="portfolio"  >
+        <h2 class="project" > Projects </h2>
+        <div class="boxholder">
+            <div class="picturelink">
```

Comment:
if you select the data using fetch all as mentioned above, you can do this in a loop